features/game/components/GamePanel.tsx

```diff
@@ -1,17 +1,39 @@
 import { Button } from '@/components/Elements/Button'
+import Select, { SelectOption } from '@/components/Elements/Select'
 import Panel from '@/components/Panel'
 import { Game } from '@prisma/client'
 import Link from 'next/link'
-import { useEffect, useState } from 'react'
+import { useEffect, useRef, useState } from 'react'
 import toast from 'react-hot-toast'
-import QRCode from 'react-qr-code'
 import { useSWRConfig } from 'swr'
 import { deleteGame } from '../api/deleteGame'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { deleteToast } from '@/components/Toasts'
+import { DocumentDownloadIcon } from '@heroicons/react/solid'
+import 'node-self'
+import QRCodeStyling, { FileExtension } from 'qr-code-styling'
+import QrCodeConfig from '@/features/game/components/QrCode/QrCodeConfig'
+import PdfTemplate, { containerStyle } from '@/features/game/components/QrCode/PdfTemplate'
+import html2canvas from 'html2canvas'
+import JsPdf from 'jspdf'
+
+const qrOptions: SelectOption[] = [
+  { value: 'PNG' },
+  { value: 'SVG' },
+  { value: 'PDF' },
+]
+
+const getStaticProp = async () => {
```

Comment:
`getStaticProps` is a reserved word for next js (https://nextjs.org/docs/basic-features/data-fetching/get-static-props) and might easily be misunderstood with this function. You don't need to export the requiring of the asset here, you can just use it where you need it

Reply:
fixed it in commit a9a3daa3 Did you mean it like that?

Reply:
You can use something like that:
```
const logo = await require('assets/logos/request-logo-single.svg') 
setImgUrl(logo.default.src)
```

Reply:
But if the `await` makes problems, your solution is good to go

Reply:
yes the await is the problem since the main function is not async